tests/testthat/test-list-widgets.R

```diff
@@ -9,7 +9,7 @@ test_that("list input & output widgets", {
   expect_equal(
     sort(inputs),
     sort(c("action", "checkbox", "checkGroup", "date", "dates", "file",
-           "num", "radio", "select", "slider1", "slider2", "text"))
+           "num", "radio", "select", "select-selectized", "slider1", "slider2", "text"))
   )
 
```

Comment:
See rstudio/shiny@77afd73

Reply:
Actually this was the culprit https://github.com/rstudio/shinytest/commit/eda9e7d48cbd5c6044d29160cafe70c82e8bd65c

Reply:
Actually, it's from Shiny: https://github.com/rstudio/shiny/commit/6b6058

https://github.com/rstudio/shiny/issues/2396